roles/tas_single_node/templates/manifests/trillian/trillian-logsigner.j2

```diff
@@ -32,8 +32,19 @@ spec:
           resources: {}
           terminationMessagePath: /dev/termination-log
           terminationMessagePolicy: File
+{% if tas_single_node_trillian_trusted_ca != "" %}
+      volumes:
+        - name: ca-trust
+          configMap:
+            name: trillian_trusted_ca
```

Comment:
The `trillian_trusted_ca` string is hardcoded at multiple places throughout this PR. Could we extract it into a variable in vars/main.yml to be able to easily change it if necessary?